workshop/content/basics/templates/flipping-formats-and-cleanup/_index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: 'Flipping formats and cleanup'
+date: 2021-12-22T16:27:38Z
+weight: 1000
+---
+
+### Overview
+You can write AWS CloudFormation [templates](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-formats.html) in JSON or YAML formats: you choose either one or the other depending on your preference. For more information, see [AWS CloudFormation template formats](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-formats.html).
+
+In this module, you will look into an example of switching from a JSON-formatted template into a YAML format: you will use the [cfn-flip](https://github.com/awslabs/aws-cfn-template-flip) tool as an option to easily perform this task. You will also use `cfn-flip` to perform opinionated cleanup actions for an example, input template.
+
+### Topics Covered
+By the end of this lab, you will be able to use `cfn-flip` as an option to:
+
+* Flip from JSON to YAML formats, and vice versa.
+* Perform opinionated cleanup actions on an example template.
+
+### Start Lab
+[Install](https://github.com/awslabs/aws-cfn-template-flip#installation) `cfn-flip` with `pip`:
+
+```shell
+$ pip install cfn-flip
+```
+
+Next, verify you can run `cfn-flip` by running the following command:
+
+```shell
+$ cfn-flip --version
+```
+
```

Comment:
Missing header here? These don't feel like `Install` steps

Reply:
No header missing - just following up as part of the install step.  I am adding clarifying text to the line before the command.  Thank you!

Reply:
Added clarifying text.